localization/pt/abstract-factory/README.md

```diff
@@ -0,0 +1,236 @@
+---
+layout: pattern
+title: Abstract Factory
+folder: abstract-factory
+permalink: /patterns/abstract-factory/
+categories: Creational
+language: pt
+tags:
+ - Gang of Four
+---
```

Comment:
Please have a look at the updated yaml frontmatter requirements in https://github.com/iluwatar/java-design-patterns/wiki/01.-How-to-contribute

Or you can just copy them from the main abstract-factory pattern

Reply:
I updated the yaml frontmatter.